### ISSUE TITLE
12-13-linalg proof of spectral radius vs norm of A

### ISSUE THREAD

**Issue by marwahaha** [rachelslaybaugh/NE155#15]
I had a couple questions about the proof that the spectral radius is always less than the norm of A.
1. Why do we say "all matrix norms are bounded"? This doesn't give us an upper bound on the norm, just a lower bound...
2. I think the line using "triangle inequality" as justification should be something else... In class, you mentioned Cauchy-Schwarz, but I think it's from submultiplicativity (**what a word**) of the matrix norm. I found a writeup here (on page 6) that does it, using a trick to turn the eigenvector into a matrix so we can take the matrix norm of it: http://www.math.drexel.edu/~foucart/TeachingFiles/F12/M504Lect6.pdf

happy to help wherever needed

**Comment by rachelslaybaugh** [comment]
1. Ah, maybe rewording that would be good; what I'm really saying is that the spectral radii are bounded. I'm betting that this was a direct translation from Jasmina's notes. Updating that header for clarity. 
2. Oh yeah, that is completely not the triangle inequality! Ah, and I see what you mean about Cauchy-Schwarz. I also like the proof that you pointed to as it makes it clear that this (a) holds for all eigenpairs and (b) must be true for the largest one. I think I'll use that since it seems clearer.

Thanks!

**Comment by rachelslaybaugh** [comment]
If that's clearer lmk and I'll close this.

**Comment by marwahaha** [comment]
Makes sense to me. Do you want me to update?

**Comment by marwahaha** [comment]
I see your update, thanks!
